src/main/java/no/nav/familie/integrasjoner/modiacontextholder/domene/ModiaContextHolderNyAktivBrukerDto.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.familie.integrasjoner.modiacontextholder.domene
+
+import no.nav.familie.kontrakter.felles.Fødselsnummer
+
+data class ModiaContextHolderNyAktivBrukerDto(
+    val personIdent: String,
+) {
+    init {
+        Fødselsnummer(personIdent)
```

Comment:
Denne vil kaste IllegalStateException hvis ugyldig fødselsnummer. Som igjen vil treffe den generiske handleException metoden, som vil trigge en alarm og klienten får 500

Anbefaler å catche denne feilen og returnere 400 bad request. F.eks med OppslagsException med nivå LAV, fordi man ikke trenger å telle inputfeil eller en egen exception som man skriver egen handler for i ApiExceptionHandler

Reply:
Sant. Det funker det.